iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -28,6 +28,37 @@ namespace iree_compiler {
 
 namespace {
 
+LogicalResult sanitizeSymbolNames(IREE::VM::ModuleOp &moduleOp) {
+  auto replaceIllegalChars = [](std::string &symbolName) {
+    // replace illegal characters in symbol name with "_"
+    std::replace(symbolName.begin(), symbolName.end(), '>', '_');
+    std::replace(symbolName.begin(), symbolName.end(), '<', '_');
+  };
+
+  std::string symbolName(moduleOp.getName());
+
+  // Sanitize name
+  replaceIllegalChars(symbolName);
+
+  if (failed(moduleOp.replaceAllSymbolUses(
+          StringAttr::get(moduleOp.getContext(), symbolName), moduleOp)))
+    return failure();
+
+  SymbolTable::setSymbolName(moduleOp, symbolName);
+
+  for (auto rodataOp : moduleOp.getOps<IREE::VM::RodataOp>()) {
```

Comment:
you'll want to do this with any module op that implements the symbol interface (you can change this to `getOps<SymbolOpInterface>`)

Reply:
I addressed this in #7863. This one only handles the op and module names.